recirq/quantum_chess/quantum_board_test.py

```diff
@@ -1284,3 +1305,278 @@ def test_split_capture_with_failed_measurement_outcome(board):
         probs = b.get_probability_distribution(100)
         assert_prob_about(probs, u.square_to_bit('a2'), 1, atol=0.1)
         assert_prob_about(probs, u.square_to_bit('a3'), 0, atol=0.1)
+
+
+@pytest.mark.parametrize("board", ALL_CIRQ_BOARDS)
+def test_merge_capture_to_fully_classical_position(board):
+    """Splits piece on d4 to b3 and c2, then merge back and capture piece on a1."""
```

Comment:
This doesn't capture a piece. This test is just a basic merge. The comment and function name should be changed accordingly.